superset-frontend/cypress-base/cypress/support/e2e.ts

```diff
@@ -160,7 +166,18 @@ Cypress.Commands.add('login', () => {
     url: '/login/',
     body: { username: 'admin', password: 'general' },
   }).then(response => {
-    expect(response.status).to.eq(200);
+    if (response.status === 302) {
```

Comment:
NOTE: not sure if this is required, but at some point was getting some redirect around login, this handles redirect as not a failure